script.js

```diff
@@ -0,0 +1,58 @@
+function selectop() {
+    var operval = document.getElementById("opervalue").value
+    switch (operval) {
+        case '+':
+            sum();
+            break;
+        case "-":
```

Comment:
You've got a lot of mixing of single and double quotes, consistency is the key to clean code.